content/en/universal_service_monitoring/guide/using_usm_metrics.md

```diff
@@ -85,22 +85,20 @@ To create an SLO from the Service Catalog:
 {{< img src="universal_service_monitoring/guide/service_catalog_slo_setup.png" alt="Setting up a Universal Service Monitoring SLO for BITSBOUTIQUE" style="width:100%;" >}}
 
 Optionally, to create an SLO manually using USM metrics:
-
-1. Navigate to **Monitors** > **SLOs** and click [**New SLO**][15].
+ 
+1. Navigate to **Service Management** > **SLOs** and click [**New SLO**][15].
 2. Select **Metric Based** and create two queries in the **Good events (numerator)** section:
    
-   * Query A: Enter a USM metric such as `universal.http.server`, an `env` primary tag in the `from` field, and select `service` from the `all values` dropdown menu. 
-   * Query B: Enter a USM metric such as `universal.http.server`, an `env` primary tag and an `error:true` tag in the `from` field, and select `service` from the `all values` dropdown menu. 
+   * Query A: Enter a USM metric such as `universal.http.server`, an `env` primary tag in the `from` field, and select `service` to the right of `all values` to scope the SLO to an individual service. 
```

Comment:
for now, we recommend filtering to a specific service, instead of grouping by service

Reply:
```suggestion
   * Query A: Enter a USM metric such as `universal.http.server`, an `env` primary tag in the `from` field, and filter the SLO to a specific `service` to the right of `all values`. 
```